tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "target": "ES2022",
+    "module": "Node16",                                
+    "rootDir": "./",     
+    "moduleResolution": "Node16",              
+    "outDir": "./dist",
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,                               
+    "skipLibCheck": true
+  },
+  "include": [
+    "./*.ts",
+    "./src/**/*.ts",
+    "./test/**/*.ts"
+  ],          
+  "exclude": ["node_modules"]
```

Comment:
Do we also need to exclude "dist"? Because it's in the exclude list in the integration-node-library tsconfig.json file.

Reply:
I found that it's a good practice to exclude the /dist folder